src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -0,0 +1,430 @@
+ *
+ * <p>
+ * This tool is designed to extract fields from the VCF to a table format that is more convenient to work with in
+ * downstream analyses.</p>
+ *
+ * <p>The user specifies one or more
+ * fields to print with the -F NAME, each of which appears as a single column in
+ * the output file, with a header named NAME, and the value of this field in the VCF
+ * one per line.  NAME can be any standard VCF column (CHROM, ID, QUAL) or any binding
+ * in the INFO field (AC=10).  In addition, there are specially supported values like
+ * EVENTLENGTH (length of the event), TRANSITION (for SNPs), HET (count of het genotypes),
+ * HOM-REF (count of homozygous reference genotypes), HOM-VAR (count of homozygous variant
+ * genotypes), NO-CALL (count of no-call genotypes), TYPE (the type of event), VAR (count of
+ * non-reference genotypes), NSAMPLES (number of samples), NCALLED (number of called samples),
+ * GQ (from the genotype field; works only for a file with a single sample), and MULTI-ALLELIC
+ * (is the record from a multi-allelic site).  </p>
+ *
+ * </p>
+ *
+ * <h3>Input</h3>
+ * <p>
+ * <ul>
+ *     <li>A VCF file</li>
+ *     <li>A list of -F fields to write</li>
+ * </ul>
+ * </p>
+ *
+ * <h3>Output</h3>
+ * <p>
+ * A tab-delimited file containing the values of the requested fields in the VCF file
+ * </p>
+ *
+ * <h3>Usage example</h3>
+ * <pre>
+ *     ./gatk-launch \
+ *     -R reference.fasta
+ *     -T VariantsToTable \
+ *     -V file.vcf \
+ *     -F CHROM -F POS -F ID -F QUAL -F AC \
+ *     -o results.table
+ * </pre>
+ * <p>would produce a file that looks like:</p>
+ * <pre>
+ *     CHROM    POS ID      QUAL    AC
+ *     1        10  .       50      1
+ *     1        20  rs10    99      10
+ *     et cetera...
+ * </pre>
+ *
+ * <h3>Caveat</h3>
+ * <p>If a VCF record is missing a value, then the tool by default throws an error, but the special value NA can
+ * be emitted instead if requested at the command line using --allowMissingData.</p>
+ *
+ * @author Mark DePristo
+ * @since 2010
+ */
+@CommandLineProgramProperties(
+        summary = "This tool is designed to extract fields from the VCF to a table format " +
+                "that is more convenient to work with in " +
+                "downstream analyses",
+        oneLineSummary = "Extract specific fields from a VCF file to a tab-delimited table",
+        programGroup = VariantProgramGroup.class
+)
+public final class VariantsToTable extends VariantWalker {
+
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc="File to which results should be written")
+    private String out = null;
+
+    /**
+     * -F NAME can be any standard VCF column (CHROM, ID, QUAL) or any binding in the INFO field (e.g., AC=10).
+     * Note that to capture GENOTYPE (FORMAT) field values, see the GF argument.  This argument accepts any number
```

Comment:
"Note that to capture GENOTYPE (FORMAT) field values, see the GF argument" is terrible grammar. Rephrase to "To capture GENOTYPE (FORMAT) field values, see the -GF argument".